WalletWasabi.Fluent/Views/Wallets/WalletView.axaml

```diff
@@ -73,8 +73,8 @@
                 <FlyoutBase.AttachedFlyout>
                   <Flyout ShowMode="Transient" Placement="Left" FlyoutPresenterClasses="Bubble">
                     <StackPanel Spacing="6">
-                      <TextBlock FontWeight="SemiBold" TextAlignment="Left">Your Bitcoin wallet is now an Anything wallet!</TextBlock>
```

Comment:
Can someone tell me what's an `Anything wallet`?

Reply:
I think it was a pun from the marketing team. ACK for changing.